BoxingSchedulerServer/functions/silently-notify-ios-app/README.md

```diff
@@ -0,0 +1,51 @@
+# Silently Notify iOS App
+
+## Prerequisites
+
+### Google Cloud Scheduler
+
+This project requires the `gcloud` command, pointed at a Google Cloud project
+with a Google Cloud Pub/Sub topic `cron-topic` and a Google Cloud Scheduler job
```

Comment:
We spoke offline that the topic and job could be created in code too. Not out of band, in this manual instruction. The topic and job are fairly integral to how the feature works. Perhaps in a follow-up PR.

Reply:
#2